Canned responses Added

NigamLad opened the pull request.

benmwilson committed:
minor additions

added a couple new phrases and changed the page title

benmwilson committed:
organised + added simple chatbox

organised files into folders and added the start of the chatbox

benmwilson committed:
added lexicon.js

This file is the Lexicon of the bot. It checks user messages against these prompts and sends back a corresponding response.

Time-Vault committed:
Update lexicon.js

Improved readability of file to make adding responses easier.

benmwilson committed:
Added index.js, assets folder, updated main.css & index.html

Front end chat box interface for bot, meaning we have the front end and back end 'speaking' to some degree.

Time-Vault committed:
Update lexicon.js

NigamLad committed:
Added more responses

Time-Vault committed:
Update lexicon.js

benmwilson commented:
We don't want to merge dev with main yet, the bot is still stupid and only repeats what the user says. @NigamLad

Time-Vault committed:
Merge branch 'Mack-Input/Output-Pairs' into dev

Time-Vault committed:
Update lexicon.js

Time-Vault committed:
Update index.js

benmwilson committed:
added delay before bot response

added some delay before the bot responds to a message. this allows the user to send a couple messages before the bot responds and the bot will always send messages in order. I think this is a good solution for now but we can revisit later.

benmwilson committed:
Update README.md

NigamLad committed:
Added more content

NigamLad committed:
Added thinking function

Does not work yet, but doesnt break the code

benmwilson committed:
added intro message

added a simple introduction message for the bot to send when the window is loaded.

benmwilson committed:
Merge remote-tracking branch 'origin/dev' into dev

NigamLad committed:
Merge branch 'dev' of https://github.com/prophet-ai/ApostleAI into dev

NigamLad committed:
Merge branch 'dev' of https://github.com/prophet-ai/ApostleAI into dev

benmwilson committed:
styled the message box and page

styled the message box to look like text messages, switch the order so the user is on the right and bot is on the left, styled the input box and buttons. styled the title and background.

NigamLad committed:
More quotes and responses

NigamLad committed:
Improved readability of front page quotes

NigamLad committed:
Fixed delayed intro bug

NigamLad committed:
Added code for specifying custom delay or calculate delay automatically

NigamLad committed:
Revert "Added code for specifying custom delay or calculate delay automatically"

This reverts commit 9ffe795ffa1eed0a8944357ae0a39f562a6eb1a1.

NigamLad committed:
Added code for Specified or calculated delay for bot messages

Comments above sendBotMessage() function explains the functioning of the new parameter

Time-Vault committed:
Update index.html

Time-Vault committed:
Update index.js

NigamLad committed:
Fixed Variable response delay
